src/App.js

```diff
@@ -1,10 +1,10 @@
-import Calculator from './Components/calculator';
+/* eslint-disable */
+import Calculator from './components/calculator';
```

Comment:
Great Job implementing the requirement :clap: , However it's a bad practise to disable any linter as Linters are useful as they automatically analyze code for potential errors, bugs, and style issues, helping us to ensure code quality and consistency so i request you to enable it also currently your app is giving a compile error see :point_down: 
![image](https://user-images.githubusercontent.com/97521557/230655382-8d7587a8-3cde-46da-aced-0a6a7911051d.png)

you can resolve it by correcting the spelling mistake in `import Calculator from './components/calculator';` and change it to `import Calculator from './Components/calculator';` since your Component folder starts with a capital C.

Reply:
Thank you so much